# Lighting Choices in Bedrooms

In [27]:
# import libraries 
import pandas as pd

In [23]:
surveydf = pd.read_csv("lightingChoices.csv") 

## Looking at the "Who" of the survey 

### Age Range

In [25]:
age = surveydf['What is your age range? ']

age_percent = age.value_counts(normalize=True) * 100
age_percent = age_percent.round(2)

age_percent

What is your age range? 
21-23    72.73
27+      13.64
18-20     9.09
24-26     4.55
Name: proportion, dtype: float64

According to the survey results, 72.73% of respondents were between the ages 21 and 23, 13.64% of the respondents were above the age of 27, 9.09% of respondents were between the ages of 18and 20, and 4.55% of the respondents were between the ages of 24 and 26. 

### Year in College 

In [26]:
year = surveydf['What year are you in college? ']

year_percent = year.value_counts(normalize=True) * 100
year_percent = year_percent.round(2)

year_percent

What year are you in college? 
Senior               63.64
Graduate Student      9.09
Junior                9.09
already graduated     4.55
Freshman              4.55
Lecturer              4.55
Graduated             4.55
Name: proportion, dtype: float64

According to the survey results, most respondents were seniors in college, accounting for 63.64% of participants. Graduate students and juniors each represented 9.09% of the sample. Individuals who had already graduated from college made up a combined 9.10% of respondents. Smaller proportions of participants identified as freshmen (4.55%) or lecturers (4.55%).

### Field of Study 

In [31]:
studyField = surveydf['What is your primary field of study/work?']

# Clean up the data 
def clean_field(field):
    if pd.isna(field):
        return field
    field = field.lower()
    
    if "stem" in field:
        return "STEM"
    elif "fine arts" in field or "design" in field or "music" in field:
        return "Fine Arts / Design"
    elif "human sciences" in field or "education" in field or "psychology" in field:
        return "Human Sciences"
    elif "social sciences" in field:
        return "Social Sciences"
    elif "business" in field or "finance" in field:
        return "Business and Finance"
    elif "humanities" in field:
        return "Humanities"
    else:
        return "Other"


# calculate percents
cleanedStudyField = studyField.apply(clean_field)

studyField_percent = (
    cleanedStudyField
    .value_counts(normalize=True)
    .mul(100)
    .round(2)
)

studyField_percent

What is your primary field of study/work?
STEM                    63.64
Fine Arts / Design      13.64
Human Sciences           9.09
Business and Finance     4.55
Social Sciences          4.55
Humanities               4.55
Name: proportion, dtype: float64

According to the survey results, 50.00% of respondents identified their primary field of study or work as STEM. Additionally, 13.64% of respondents were in Human Sciences, 9.09% of respondents were in Fine Arts or Design, 9.09% of respondents were in Social Sciences, 4.55% of respondents were in Business and Finance, and 4.55% of respondents were in Humanities.

### Gender Identity 

In [ ]:
gender = surveydf['What is your gender identity? ']

gender_percent = gender.value_counts(normalize=True) * 100
gender_percent = gender_percent.round(2)

gender_percent

What is your gender identity? 
Woman          50.00
Man            45.45
Genderqueer     4.55
Name: proportion, dtype: float64

According to the survey results, 50.00% of the respondents identified as Women. Additionally, 45.45% of the respondents identified as Male, and 4.55% of respondents were Genderqueer. 

### Housing

In [33]:
housing = surveydf['What type of housing do you live in? ']

housing_percent = housing.value_counts(normalize=True) * 100
housing_percent = housing_percent.round(2)

housing_percent

What type of housing do you live in? 
On-campus apartment/suite/townhouse    40.91
On-campus traditional-style dorm       22.73
Off-campus house                       18.18
With family                            13.64
Off-campus apartment                    4.55
Name: proportion, dtype: float64

According to the survey results, 40.91% of respondents reported living in an on-campus apartment, suite, or townhouse. Additionally, 22.73% of respondents lived in a traditional-style on-campus dorm. Off-campus housing was less common, with 18.18% of respondents living in an off-campus house and 4.55% living in an off-campus apartment. Finally, 13.64% of respondents reported living with family.

### Shared Bedroom

In [34]:
sharedBedroom = surveydf['Do you currently share a bedroom with someone? ']

sharedBedroom_percent = sharedBedroom.value_counts(normalize=True) * 100
sharedBedroom_percent = sharedBedroom_percent.round(2)

sharedBedroom_percent

Do you currently share a bedroom with someone? 
No           50.00
Yes          45.45
Sometimes     4.55
Name: proportion, dtype: float64

According to the survey results, 50.00% of respondents reported that they do not currently share a bedroom with someone. In contrast, 45.45% of respondents indicated that they do share a bedroom, while a smaller portion of respondents (4.55%) reported that they sometimes share a bedroom.

## Looking at the "Where" of the survey 

### Natural Lighting in Room 

In [35]:
naturalLighting = surveydf['How much natural light does your room get? ']

naturalLighting_percent = naturalLighting.value_counts(normalize=True) * 100
naturalLighting_percent = naturalLighting_percent.round(2)

naturalLighting_percent

How much natural light does your room get? 
A lot of natural light           50.00
Some natural light               31.82
Very little natural light        13.64
No windows / no natural light     4.55
Name: proportion, dtype: float64

According to the survey results, half of respondents (50.00%) reported that their room receives a lot of natural light. Additionally, 31.82% of respondents indicated that their room receives some natural light. Fewer respondents reported limited exposure to natural light, with 13.64% stating their room receives very little natural light, and 4.55% reporting that their room has no windows or access to natural light.

## Looking at the "What" in the survey 

### Lighting Choices 

In [36]:
lightingChoices = surveydf['What types of lighting do you regularly use in your room? ']

lightingChoices_percent = lightingChoices.value_counts(normalize=True) * 100
lightingChoices_percent = lightingChoices_percent.round(2)

lightingChoices_percent

What types of lighting do you regularly use in your room? 
Overhead ceiling light, Desk lamp                                    18.18
Desk lamp, Floor lamp, String lights                                  9.09
Desk lamp                                                             9.09
Floor lamp                                                            9.09
Desk lamp, Floor lamp                                                 9.09
LED strip lights                                                      4.55
LED strip lights, Floor lamp, String lights                           4.55
String lights                                                         4.55
Desk lamp, String lights                                              4.55
LED strip lights, Desk lamp, I mostly keep lights off                 4.55
LED strip lights, Desk lamp, Floor lamp, I mostly keep lights off     4.55
Overhead ceiling light, LED strip lights, Desk lamp                   4.55
Desk lamp, Himalayan salt lamp           

Survey responses indicate that most respondents rely on a combination of lighting sources rather than a single type. Desk lamps were the most common lighting feature, often used alone or in combination with floor lamps, overhead ceiling lights, string lights, or LED strip lights. Overhead ceiling lights and floor lamps were also frequently mentioned, while decorative lighting such as string lights, LED strip lights, and Himalayan salt lamps appeared less commonly. A small number of respondents reported that they mostly keep their lights off.

### Intensity of lighting

In [37]:
lightingIntense = surveydf['What is the intensity of your main source of lighting? ']

lightingIntense_percent = lightingIntense.value_counts(normalize=True) * 100
lightingIntense_percent = lightingIntense_percent.round(2)

lightingIntense_percent

What is the intensity of your main source of lighting? 
Warm white/yellow                                    77.27
White                                                13.64
I use LED lights/strip mainly and can change them     9.09
Name: proportion, dtype: float64

According to the survey results, the majority of respondents (77.27%) reported that their main source of lighting is warm white or yellow. A smaller portion of respondents indicated using white lighting (13.64%), while 9.09% reported primarily using adjustable LED lights or light strips that allow them to change the lighting intensity or color.

### LED Strip color Prefrence 

In [40]:
ledColor = surveydf['If you use LED lights/strip as your main source of lighting, what color do you prefer during the day? ']

ledColor_percent = ledColor.value_counts(normalize=True) * 100
ledColor_percent = ledColor_percent.round(2)

ledColor_percent

If you use LED lights/strip as your main source of lighting, what color do you prefer during the day? 
I do not use LED lights/strip as my main source of lighting in my room.    86.36
Purple                                                                      4.55
Orange                                                                      4.55
Blue                                                                        4.55
Name: proportion, dtype: float64

According to the survey results, the majority of respondents (86.36%) reported that they do not use LED lights or light strips as their main source of lighting during the day. Among those who do use LED lighting, preferences were evenly split, with 4.55% of respondents each preferring purple, orange, or blue lighting during the day.

## Looking at the "When" in the survey 

#### Sleeping

In [41]:
sleep = surveydf['Do you sleep with any lights on?']

sleep_percent = sleep.value_counts(normalize=True) * 100
sleep_percent = sleep_percent.round(2)

sleep_percent

Do you sleep with any lights on?
No - I sleep in the dark    86.36
Yes - String lights          4.55
Yes - LED strip lights       4.55
Yes - lamp                   4.55
Name: proportion, dtype: float64

According to the survey results, the vast majority of respondents (86.36%) reported that they sleep with no lights on, preferring complete darkness. A small portion of respondents indicated sleeping with some form of lighting, with 4.55% each reporting the use of string lights, LED strip lights, or a lamp while sleeping.

### Occurence of Overhead Lighting

In [42]:
overheadLighting = surveydf['How often do you use your overhead ceiling light? ']

overheadLighting_percent = overheadLighting.value_counts(normalize=True) * 100
overheadLighting_percent = overheadLighting_percent.round(2)

overheadLighting_percent

How often do you use your overhead ceiling light? 
Rarely                             40.91
Sometimes                          22.73
Very often                         18.18
Never                              13.64
I do not have an overhead light     4.55
Name: proportion, dtype: float64

According to the survey results, 40.91% of respondents reported that they rarely use their overhead ceiling light. Additionally, 22.73% indicated that they sometimes use it, while 18.18% reported using their overhead light very often. A smaller portion of respondents reported never using an overhead ceiling light (13.64%), and 4.55% indicated that they do not have an overhead light in their room.

### Overhead vs. Underhead occurence 

In [45]:
occurence = surveydf['When do you mostly use LED strips or lamps instead of overhead lighting?']

occurence_percent = occurence.value_counts(normalize=True) * 100
occurence_percent = occurence_percent.round(2)

occurence_percent

When do you mostly use LED strips or lamps instead of overhead lighting?
I do not use LED strips or lamps           36.36
Afternoon, Evening, Late night             18.18
Late night                                 13.64
Morning, Afternoon, Evening, Late night    13.64
Morning, Evening, Late night                9.09
Evening                                     4.55
Morning                                     4.55
Name: proportion, dtype: float64

According to the survey results, 36.36% of respondents reported that they do not use LED strips or lamps in place of overhead lighting. Among those who do, usage was most common later in the day, with 18.18% reporting use during the afternoon, evening, and late night, and 13.64% indicating use primarily during late night hours. Some respondents reported using alternative lighting throughout most of the day (13.64%), while smaller portions reported use in the morning and evening (9.09%), exclusively in the evening (4.55%), or only in the morning (4.55%).

## Looking at the "Why" in the survey

### Preference 

In [46]:
preference = surveydf['Why do you prefer LED strips or lamps over overhead lighting?']

preference_percent = preference.value_counts(normalize=True) * 100
preference_percent = preference_percent.round(2)

preference_percent

Why do you prefer LED strips or lamps over overhead lighting?
Softer lighting, Looks better aesthetically, Overhead light is too bright                                         22.73
I do not prefer LED strips or lamps                                                                               18.18
Softer lighting, Looks better aesthetically, Helps me relax, Better for studying, Overhead light is too bright     9.09
Softer lighting, Looks better aesthetically, Helps me relax, Overhead light is too bright                          9.09
Softer lighting, Looks better aesthetically, Better for studying, Overhead light is too bright                     9.09
Softer lighting, Looks better aesthetically, Helps me relax, Better for studying                                   9.09
Helps me relax                                                                                                     4.55
Softer lighting                                                                                   

According to the survey results, respondents who prefer LED strips or lamps over overhead lighting most commonly cited softer lighting and improved aesthetics as key reasons, often noting that overhead lighting feels too bright. Many respondents also indicated that alternative lighting helps them relax and creates a better environment for studying. In contrast, 18.18% of respondents reported that they do not prefer LED strips or lamps over overhead lighting.

### Mood and Productivity

In [47]:
moodProductivity = surveydf['Do you feel your lighting setup affects your mood or productivity?']

moodProductivity_percent = moodProductivity.value_counts(normalize=True) * 100
moodProductivity_percent = moodProductivity_percent.round(2)

moodProductivity_percent

Do you feel your lighting setup affects your mood or productivity?
Agree             45.45
Strongly agree    40.91
Neutral           13.64
Name: proportion, dtype: float64

According to the survey results, the majority of respondents feel that their lighting setup affects their mood or productivity, with 45.45% agreeing and 40.91% strongly agreeing. A smaller portion of respondents (13.64%) reported feeling neutral on the matter.

## Methods

This survey was distributed online using Google Forms and shared with participants through the class Discord, with fellow college students, and with friends and family. Participants completed the survey voluntarily and anonymously. Respondents were given approximately five days to complete the survey. The survey consisted primarily of multiple-choice and checkbox questions related to lighting preferences and habits in their rooms.

Data analysis was performed using Python in a Jupyter Notebook. The pandas library was used for data cleaning, grouping, and calculating percentages, while Matplotlib was used to create visualizations for each question to clearly display trends and distributions.

## Limitations

This study relied on self-reported data, which may introduce response bias. Additionally, the sample size was relatively small and consisted mainly of college students, limiting generalizability. Some questions allowed open-ended responses or had inconsistent wording, which required manual cleaning and grouping for analysis.

Future studies could increase the sample size, provide more standardized answer options to reduce variability, and collect responses from a broader population to improve representativeness.

## Discussion and Conclusion

Several trends emerged from the survey results. Most participants preferred LED strips or 
lamps over overhead lighting, especially during the evening and late-night hours. Softer 
lighting and aesthetic appeal were the most common reasons for this preference.

A large portion of participants reported that their lighting setup affects their mood or 
productivity, suggesting that lighting plays an important role in personal spaces. 
However, some categories showed no strong patterns due to varied individual preferences.

Overall, the results indicate a clear shift away from overhead lighting toward customizable 
and ambient lighting options among society. 